### PR TITLE
ibmcloud: add page

### PR DESCRIPTION
**Pull request by Holomech**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ibmcloud.md
Comment (Waples) [inline pages/common/ibmcloud.md:10]: In my opinion this is an unnessecary example.

File: pages/common/ibmcloud.md
Comment (Waples) [inline pages/common/ibmcloud.md:16]: Does this only update the `cli` tool or also other items related to IBM cloud(apps)?

Comment (Holomech) [reply pages/common/ibmcloud.md:16]: To the best of my knowledge (and according to official docs) this updates only the CLI. I can add that for clarity.

File: pages/common/ibmcloud.md
Comment (Holomech) [reply pages/common/ibmcloud.md:10]: OK, I can see why. I based this on the git page, as recommended in the CONTRIBUTING document.

Comment (marchersimon) [reply pages/common/ibmcloud.md:10]: Depending on the command, it can be useful to know which version you are using. We have a `version` exaple in many pages. However we always keep it at the botton

File: pages/common/ibmcloud.md
Comment (marchersimon) [inline pages/common/ibmcloud.md:4]: <https://cloud.ibm.com/docs/cli?topic=cli-ibmcloud_cli> actually covers all of the command line options I think.

File: pages/common/ibmcloud.md
Comment (marchersimon) [inline pages/common/ibmcloud.md:6]: Like `version`, we keep `help` examples always at the bottom.

File: pages/common/ibmcloud.md
Comment (marchersimon) [inline pages/common/ibmcloud.md:14]:
```suggestion
- Update `ibmcloud` to the latest version:
```

**Review by marchersimon — COMMENTED** [review]
Thanks @Holomech

File: pages/common/ibmcloud.md
Comment (Holomech) [reply pages/common/ibmcloud.md:4]: Yes, you're right, I had some trouble finding reference for the general command in the docs. This is a way better link. The one I put in was from the CLI help page :-)

File: pages/common/ibmcloud.md
Comment (marchersimon) [reply pages/common/ibmcloud.md:16]: I think when adding ``` ` ... ` ``` around it like we do in all pages when talking about commands, it's more clear that you're talking about the CLI itself.

**Review by marchersimon — APPROVED** [review]

File: pages/common/ibmcloud.md
Comment (navarroaxel) [inline pages/common/ibmcloud.md:22]: I like the idea to split this into `Display help` and `Display help for a subcommand`

Comment (Holomech) [reply pages/common/ibmcloud.md:22]: Yeah, I was thinking about this as well, I just thought it might be too verbose.

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — COMMENTED** [review]
Approved.  I'm sorry, I didn't get to this (you guys are too fast!)

**Comment by bl-ue** [comment]
@Waples in the future, will you please approve before you merge, in addition to the other approvals?

Also, when someones requests someone else's review, unless that person is not likely to respond (e.g. one of our less active members) you can approve, but don't merge, because they may have comments ;)

**Comment by bl-ue** [comment]
Thank you @Holomech and welcome to tldr-pages!  Out of curiosity: are you planning to add pages for any `ibmcloud` subcommands?

**Comment by Holomech** [comment]
Thank you @bl-ue, I'm a newbie, as this was my first pull request for an open-spurce project on GitHub. I hope I did an OK job. I do want to add some subcommands for ibmcloud, at least the ones I've used before. I don't know, how much time I'll have in the next couple of weeks, but I'll try to write a couple of pages.

**Comment by bl-ue** [comment]
Great @Holomech! You did a great job. Please feel free to contribute in any way or amount and at any time you'd like! tldr-pages is a volunteer project and we're all just community members, like yourself.

**Comment by Waples** [comment]
> Thank you @bl-ue, I'm a newbie, as this was my first pull request for an open-spurce project on GitHub. I hope I did an OK job. I do want to add some subcommands for ibmcloud, at least the ones I've used before. I don't know, how much time I'll have in the next couple of weeks, but I'll try to write a couple of pages.

Thanks bro <3

**Pull request merged**
